src/chat.h

```diff
@@ -17,7 +17,7 @@ class ChatChannel
 {
 public:
 	ChatChannel() = default;
-	ChatChannel(uint16_t channelId, std::string channelName) : id{channelId}, name{std::move(channelName)} {}
+	ChatChannel(uint16_t channelId, std::string channelName) : m_id{channelId}, m_name{std::move(channelName)} {}
```

Comment:
I'd rather not, m_ prefix sucks

Reply:
The reason I added the prefix was because I declared a local variable with the same variable member name (id) when I was migrating it, breaking the Chat. I can keep as it was if you think it is better that way.